README.md

```diff
@@ -1,125 +1,118 @@
+> Supports full or partial version numbers, starting with an optional "v".
 
-Download and switch to the latest LTS (long-term support) Node.js release.
+The `nvm install` command activates the specified Node version only in the current environment. If you want to set the default version for new shells use:
 
-```console
-nvm use lts
+```fish
+set -U nvm_default_version v12.9.1
 ```
 
-You can create a `.nvmrc` file in the root of your project (or any parent directory) and run `nvm` to use the version in it. `nvm` will try to find the nearest `.nvmrc` file, traversing the directory tree from the current working directory upwards.
+Activate a version you've already installed.
 
 ```console
-node -v > .nvmrc
-nvm
+nvm use lts
 ```
 
-Run `nvm` in any subdirectory of a directory with an `.nvmrc` file to switch to the version from that file. Similarly, running `nvm use <version>` updates that `.nvmrc` file with the specified version.
+List which versions you have installed (including the system Node if there is one).
 
 ```console
-├── README.md
-├── dist
-├── node_modules
-├── package.json
-└── src
-    └── index.js
+$ nvm list
+     system  
+    v8.17.0 lts/carbon 
+    v12.9.1  
+ ▶ v14.15.1 lts/fermium 
+    v15.3.0 latest
 ```
 
+Or list all the Node versions available to install.
+
 ```console
-echo lts >.nvmrc
-cd src
-nvm
-node -v
-v10.15.1
+$ nvm list-remote
+   ...
+   v14.13.1  
+   v14.14.0  
+   v14.15.0 lts/fermium 
+ ▶ v14.15.1 lts/fermium ✓
+    v15.0.0  
+    v15.0.1  
+    v15.1.0  
+    v15.2.0  
+    v15.2.1  
+    v15.3.0 latest ✓
 ```
 
-### Listing versions
-
-List all the supported Node.js versions you can download and switch to.
+Want to remove a Node version? You can do that too.
 
 ```console
-nvm ls
+nvm remove v12.9.1
 ```
 
-```console
-...
-10.14.2    (lts/dubnium)
-10.15.0    (lts/dubnium)
- 11.0.0
- 11.1.0
- 11.2.0
- 11.3.0
- 11.4.0
- 11.5.0
- 11.6.0
- 11.7.0    (latest/current)
+If you would like to use a different mirror of the Node binaries, for example, if you're behind a firewall, use:
+
+```fish
+set -g nvm_mirror https://npm.taobao.org/mirrors/node
 ```
 
-You can use a regular expression to narrow down the output.
+## `.nvmrc`
 
-```console
-nvm ls '^8.[4-6]'
-```
+An `.nvmrc` file makes it easy to peg a specific version of Node for different projects. Just create an `.nvmrc` file containing a Node version number or alias, e.g., `node`, `lts`, `carbon`, etc., in the root of your project.
 
 ```console
-8.4.0    (lts/carbon)
-8.5.0    (lts/carbon)
-8.6.0    (lts/carbon)
+node -v >.nvmrc
 ```
 
-To customize the download mirror, e.g., if you are behind a firewall, you can set `$nvm_mirror`:
+Then run `nvm install` to install and activate that version. This will traverse the directory hierarchy looking for the nearest `.nvmrc` file.
```

Comment:
I would potentially move the "This will" to a new paragraph, elaborating slightly. The reason is that otherwise it sounds like the whole thing (including `node -v > .nvmrc` is needed for the traversal to take effect. (I know this might be obvious for most, but I am sure someone will stumble upon this wondering what "this" referred to.

Reply:
I'll think about it! Thanks.